src/selection.ts

```diff
@@ -1,35 +1,55 @@
 import {SingleDefChannel} from './channel';
 import {VgBinding} from './vega.schema';
 
-
 export type SelectionTypes = 'single' | 'multi' | 'interval';
 export type SelectionResolutions = 'global' | 'independent' | 'union' |
   'union_others' | 'intersect' | 'intersect_others';
 
 export interface BaseSelectionDef {
-  // domain?: SelectionDomain;
-  resolve?: SelectionResolutions;
   on?: any;
+  resolve?: SelectionResolutions;
   // predicate?: string;
-  bind?: 'scales' | VgBinding | {[key: string]: VgBinding};
+  // domain?: SelectionDomain;
```

Comment:
Because they're technically part of our model (as described in the paper) but not something we support yet in 2.0. I wanted to leave it in, but commented, to remind us should we consider them in the future.